net.h

```diff
@@ -142,4 +143,44 @@ namespace con {
 
     outputLayer->getResults(results);
   }
+
+  void testKaggle(
+  		vector<short> &kaggleResults,
+  		const int &batchSize,
+  		const vector<Layer*> &layers) {
+
+  	vector<int> results;
+  	vector<Sample> validateData;
+
+		for (int i = 0; i < 300000; i += batchSize) {
+			if (i % 10000 == 0) {
+				cout << "Validating: " << i << endl;
+			}
+
+			readKaggle(&validateData, i, batchSize);
+
+			input.clear();
```

Comment:
I can't because the 2 functions only return the number of correct guesses. I need the labels to write the kaggle results.